common/mail.js

```diff
@@ -223,19 +221,7 @@ function sendEmail({ name, mailConfig, mailTransport = null }) {
         const transport = mailTransport ? mailTransport : createSesTransport();
 
         return transport.sendMail(buildMailOptions(mailConfig)).then(info => {
-            /**
-             * Record a send count as a CloudWatch event if enabled
-             * istanbul ignore if
```

Comment:
lol at the also-redundant reference to `istanbul`.

Reply:
It's not Constantinople